hazelcast-jet-distribution/src/root/config/jvm.options

```diff
@@ -0,0 +1,49 @@
+# JVM Configuration
+
+# One option per line
+# Lines starting with # and empty lines are ignored
+
+# Memory configuration
+#-Xms4g
+#-Xmx4g
+#-XX:+AlwaysPreTouch
+
+# Generate heap dump on out-of-memory error
+# Uncomment to enable
+#-XX:+HeapDumpOnOutOfMemoryError
+#-XX:HeapDumpPath=./
+
+# Enable debugger
+#-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+# Enable JMX
+#-Dcom.sun.management.jmxremote
+#-Dcom.sun.management.jmxremote.port=9010
+#-Dcom.sun.management.jmxremote.authenticate=false
+#-Dcom.sun.management.jmxremote.ssl=false
+#-Dcom.sun.management.jmxremote.local.only=false
+
+# GC configuration
+
+# G1GC Configuration
```

Comment:
can we note here that this should be used if you using Java 8?

Reply:
Added.